### DISCUSSION TITLE
下载的稳定版本不停的闪退

### DISCUSSION THREAD

**Discussion by wniter** [getlantern/lantern-forum-cn#635] [Q&A] [answered]
下载的稳定版本不停的闪退，怎么解决

**Comment by 904787103** [comment]
没办法解决，这问题我都反馈了3天了，每次都是耐心等候，现在正式版闪退，测试版无法授权，根本没一个能用的

**Reply by ycjdixon** [reply, marked as answer]
先卸载那个所谓的“稳定版”7.8.6，然后再重新安装7.8.4或者7.8.5版本。不用去试着安装8.0.0版本了，那个版本不能用的。他们技术团队不干事的，你还得靠自己，续费的时间总要用完的。

**Comment by tena1210** [comment]
关键是7.8.5版本去哪找呀，我都找了三天了，都没找到

**Reply by ycjdixon** [reply]
@tena1210
你发我邮箱地址  我传给你

**Reply by wniter** [reply]
595436346@qq.com

**Reply by ycjdixon** [reply]
发你邮箱了

**Reply by Pollywant** [reply]
1648912798@qq.com
麻烦也发我个，谢谢了！

**Reply by ycjdixon** [reply]
@Pollywant
已经发你了。7.8.6版本闪退问题提出来那么久了，根本没有跟进过。

**Reply by MMschool** [reply]
我也需要一个，麻烦发我一个wottanker001@163.com

**Reply by ycjdixon** [reply]
@MMschool
发你了

**Reply by MMschool** [reply]
我用了7.8.5这个版本还是解决不了打开后，应用就自动关闭的问题，用最新的测试版没出现这个问题，但是无法授权

**Reply by MMschool** [reply]
下载一个win7版本可以正常在win10上面用和支付，真奇葩
